supertokens_python/recipe/emailverification/recipe.py

```diff
@@ -45,19 +49,27 @@
 class EmailVerificationRecipe(RecipeModule):
     recipe_id = 'emailverification'
     __instance = None
+    email_delivery: EmailDeliveryIngredient[TypeEmailVerificationEmailDeliveryInput]
 
     def __init__(self, recipe_id: str, app_info: AppInfo,
-                 config: ParentRecipeEmailVerificationConfig):
+                 config: ParentRecipeEmailVerificationConfig,
+                 ingredients: EmailVerificationIngredients,
```

Comment:
Ingredients can be None too? If not, then you also need to make the emailpassword ingredients as non nullable

Reply:
Made both non-nullable.